.eslintrc.js

```diff
@@ -11,7 +11,11 @@ module.exports = {
   ],
   overrides: [
     {
-      files: ['*.test.ts', '*.test.js'],
+      files: [
+        '*.test.ts',
+        '*.test.js',
+        './packages/network-controller/src/provider-api-tests/**/*.ts',
```

Comment:
Hmm, this makes me think that each package should probably have its own `.eslintrc.js` for consistency with the TypeScript config files, etc., but this is fine for now.

Reply:
agreed. Lets make a new issue for this: https://github.com/MetaMask/core/issues/1100